docs/validations.md

```diff
@@ -0,0 +1,134 @@
+# Getting started with validations, Formik and Yup
```

Comment:
Could we just update this file `carbon/deploy/docs/guides/validations.md` with this information about `validation` instead od creating a new one ?